src/integration_tests/conftest.py

```diff
@@ -60,16 +67,19 @@ def post(self, endpoint, json):
 
 @pytest.fixture
 def client():
-    api_url = os.environ.get("API_URL")
-    if api_url:
-        if os.environ.get("GOOGLE_APPLICATION_CREDENTIALS"):
+    if (
+        config.CONF == "IntegrationTestingCloud"
+        or config.CONF == "cloud-integration-test"
```

Comment:
Can CONF be either of these values? How come?

Reply:
It can be either, i think this is just a case of terrible variable naming, will change